createdb.py

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/python
 
+from __future__ import print_function
```

Comment:
Just for completeness, I'd rather keep that in there, because on Python2,
print() has different semantics unless you use __future__.
I'm OK with dropping __future__ when the whole script migrates to /usr/bin/python3 ,
but there are other more complicated things preventing that just now.

Thanks for the review.